bytes/src/lib.rs

```diff
@@ -51,15 +52,15 @@ pub mod arc {
         sequestered: Arc<dyn Any>,
     }
 
-    // Synchronization happens through `self.sequestered`, which mean to ensure that even
+    // Synchronization happens through `self.sequestered`, which means to ensure that even
     // across multiple threads each region of the slice is uniquely "owned", if not in the
     // traditional Rust sense.
-    unsafe impl Send for Bytes { }
+    unsafe impl Send for BytesMut { }
```

Comment:
Consider removing the `Send` implementation for `BytesMut`. Timely compiles and tests without it.

Reply:
Great point! I also added some notes about things that are not implemented but could be, if we wanted a more broadly usable crate (for the moment, no such goal).